### PR TITLE
Update regress2sum to remove contrib.learn dependency

### PR DESCRIPTION
**Pull request by ymt123**
Updating to remove dependency on tf.contrib.learn which changed/no longer seems to work in Tensorflow v10.

### CODE REVIEW

**Comment by ymt123** [comment]
The return of np.zeros(200) in tags_2_vec is pretty hacky but that should go away in @kylemvz's changes to incorporate the text transformer

**Comment by kylemvz** [comment]
Is 1.001 a better default learning rate than 0.5?

**Comment by kylemvz** [comment]
Also, the docstring for `train_model` still references the deprecated save_path arg (I think you replaced it with model_input_path and model_output_path)

**Comment by ymt123** [comment]
@kylemvz It's not clear to me what the right default learning rate is (~1 should be fine for at least iaprtc). I've updated several of the comments to reflect that changes
